docs/release-notes.rst

```diff
@@ -1,6 +1,46 @@
 Release Notes
 =============
 
+Restraint 0.2.1
+---------------
+
+Upgrades:
+~~~~~~~~~
+* | Provide ease of use with Restraint commands
+  | When running commands outside of jobs on the local host,
+    some `restraint` commands require manually setting up
+    environment variables or constructing long URLs
+    before running. This can be issue if you are trying
+    to extend the watchdog in a timely fashion.
+    A new option was added which requires the
+    argument `--port <restraint-port-number>`.
+    Commands affected are rstrnt-report-log,
+    rstrnt-report-result, rstrnt-abort, and
+    rstrnt-watchdog.
+* | :bug:`1802610`: Document how to remove RHTS from Jobs
+  | Added new section :ref:`rm_rhts_guide` detailing
+    substitutes for RHTS scripts, environment variables,
+    and testinfo.desc file and associated variables.
+
+Bug Fixes:
+~~~~~~~~~~
+* | Revert process stdin behavior
+  |  Daniel to provide input.
```

Comment:
Fix: Redirect task STDIN back to /dev/null
In release 0.2.0 the task STDIN was redirected to a pipe shared with the server. This breaks ausearch command when the input is not explicitly specified, as by default, if STDIN is a pipe, it will read from it, instead of system logs. As the pipe is closed when the task is running, tests expecting matches failed, and tests expecting no matches were unreliable.
Restoring redirect of task STDIN back to /dev/null ensures that ausearch reads from system logs by defautl.